src/components/Chat/InputMessageV2.tsx

```diff
@@ -422,15 +419,15 @@ const InputMessageV2 = ({
           onFocus={() => setInputFocus(true)}
           onBlur={() => setInputFocus(false)}
           numberOfLines={4}
+          keyboardAppearance="dark"
         />
         {isShowToggle && (
           <ToggleSwitch
             labelLeft={
               !inputFocus ? (isAnonimityEnabled ? 'Incognito' : 'Incognito disabled') : undefined
             }
             styleLabelLeft={[
-              styles.labelToggle,
-              {color: type === 'ANONYMOUS' ? COLORS.gray : COLORS.blue}
+              {color: type === 'ANONYMOUS' ? COLORS.anon_primary : COLORS.signed_primary}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `keyboardAppearance="dark"` is fine if it fits the app's theme. The changes in the `ToggleSwitch` component seem to be related to color and text updates. Make sure these changes align with the overall design and provide a consistent user experience.

```diff
-          styles.labelToggle,
-          {color: type === 'ANONYMOUS' ? COLORS.gray : COLORS.blue}
+          {color: type === 'ANONYMOUS' ? COLORS.anon_primary : COLORS.signed_primary}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->